components/compositing/compositor.rs

```diff
@@ -667,14 +654,9 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
                 }
             },
 
-            (CompositorMsg::Forwarded(msg), ShutdownState::NotShuttingDown) => {
+            CompositorMsg::Forwarded(msg) => {
                 self.handle_webrender_message(msg);
             },
-
-            // When we are shutting_down, we need to avoid performing operations
-            // such as Paint that may crash because we have begun tearing down
-            // the rest of our resources.
```

Comment:
I think this comment might be invalid now. It refers to `Paint` which is a pre-WebRender concept as far as my memory serves. The main issue here is that I think we don't want to keep compositing because the other parts of Servo are in a transient state.